src/pages/ProvideLiquidity/LiquidityCard.tsx

```diff
@@ -56,39 +61,90 @@ export default function LiquidityCard({
     setExpanded((prev) => !prev)
   }
 
+  const handleSetActionType = (type: ACTION_TYPE) => {
+    setActionType(type)
+  }
+
   if (!token || !rfpToken) {
     return null
   }
 
   return (
-    <PoolCard
-      token0={token}
-      token1={rfpToken}
-      poolTitle={t('liquidityCardTitle', { token0: farmBotSummary.token0Name, token1: farmBotSummary.token1Name })}
-      buttonLabel={t('addLiquidity')}
-      buttonOnPress={handleAddLiquidity}
-      buttonActive={expanded}
-      tvlCUSD={tvlCUSD}
-      tvlCUSDInfo={t('farmBotTVLInfo')}
-      PoolDetails={PoolDetails}
-    >
-      <Container $expanded={expanded}>
-        <AddLiquidityConfirm
-          token0={token}
-          token1={rfpToken}
-          isOpen={showConfirm}
-          onDismiss={() => {
-            setShowConfirm(false)
-          }}
-        />
-        <AddLiquidityForm
-          token0={token}
-          token1={rfpToken}
-          onConfirmAddLiquidity={() => {
-            setShowConfirm(true)
-          }}
-        />
-      </Container>
-    </PoolCard>
+    <>
+      <PoolCard
+        token0={token}
+        token1={rfpToken}
+        poolTitle={t('liquidityCardTitle', { token0: farmBotSummary.token0Name, token1: farmBotSummary.token1Name })}
+        buttonLabel={userBalance ? t('manage') : t('addLiquidity')}
+        buttonOnPress={handleAddLiquidity}
+        buttonActive={expanded}
+        tvlCUSD={tvlCUSD}
+        tvlCUSDInfo={t('farmBotTVLInfo')}
+        PoolDetails={PoolDetails}
+      >
+        <Container $expanded={expanded}>
+          {userBalance && (
+            <RowBetween marginTop="10px">
+              <ButtonPrimary
+                onClick={() => handleSetActionType('add')}
+                padding="8px"
+                borderRadius="8px"
+                width="48%"
+                inverse={actionType !== 'add'}
+              >
+                {t('addLiquidity')}
+              </ButtonPrimary>
+              <ButtonPrimary
+                onClick={() => handleSetActionType('remove')}
+                padding="8px"
+                borderRadius="8px"
+                width="48%"
+                inverse={actionType !== 'remove'}
+              >
+                {t('removeLiquidity')}
+              </ButtonPrimary>
+            </RowBetween>
+          )}
+          {actionType == 'add' && (
+            <AddLiquidityConfirm
+              token0={token}
+              token1={rfpToken}
+              isOpen={showConfirmAdd}
+              onDismiss={() => {
+                setShowConfirmAdd(false)
+              }}
+            />
+          )}
+          {actionType == 'add' && (
+            <AddLiquidityForm
+              token0={token}
+              token1={rfpToken}
+              onConfirmAddLiquidity={() => {
+                setShowConfirmAdd(true)
+              }}
+            />
+          )}
```

Comment:
```suggestion
          {actionType == 'add' && (
            <AddLiquidityConfirm
              token0={token}
              token1={rfpToken}
              isOpen={showConfirmAdd}
              onDismiss={() => {
                setShowConfirmAdd(false)
              }}
            />
          <AddLiquidityForm
              token0={token}
              token1={rfpToken}
              onConfirmAddLiquidity={() => {
                setShowConfirmAdd(true)
              }}
            />
          )}
```
I didn't actually test this but it seems cleaner

Reply:
you can do the same with the 'remove' part below